client/src/styles/legacy/GridField.scss

```diff
@@ -269,7 +270,7 @@
             &.hover-alike,
             &:hover {
               color: $white;
-              background-color: darken($gray-dark, 10);
+              background-color: darken(map-get($colors, gray-dark), 10);
```

Comment:
rather than looking up the map, can just replace with `$gray-700` like it was done everywhere else